example/image-classification/test_score.py

```diff
@@ -0,0 +1,46 @@
+"""
+test pretrained models
+"""
+import argparse
+import mxnet as mx
+from common import find_mxnet, modelzoo
+from common.util import download_file
+from score import score
+
+def download_data():
+    download_file('http://data.mxnet.io/data/val-5k-256.rec', 'data/val-5k-256.rec')
+
+def test_imagenet1k_resnet(args):
+    models = ['imagenet1k-resnet-34',
+              'imagenet1k-resnet-50',
+              'imagenet1k-resnet-101',
+              'imagenet1k-resnet-152']
+    accs = [.72, .75, .765, .76]
+    for (m, g) in zip(models, accs):
+        acc = mx.metric.create('acc')
+        (speed,) = score(model=m, data_val='data/val-5k-256.rec',
+                         rgb_mean='0,0,0', metrics=acc, **vars(args))
```

Comment:
The goal is to be consistent. Currently all other models except resnet need mean substraction. In the short term we should modify resnet to be consistent with the other models. In the long term we should retrain all the other models to make a batchnorm layer as first layer standard.

Note that mean substraction doesn't exclude using bn on the first layer, it will have no effect. This is the best solution I can think of that minimize confusion.